### DISCUSSION TITLE
How to replace `createMemoryHistory` using the `react-router-dom-v5-compat` package?

### DISCUSSION THREAD

**Discussion by meseck** [remix-run/react-router#9970] [v5 to v6 Migration]
Hey everyone! 👋 

I'm trying to write some tests for a route that already uses v6 with `react-router-dom-v5-compat`. In the codebase I am currently working in, there are tests that use `createMemoryHistory`, but this function doesn't seem to be used in v6 anymore. This [stackoverflow answer](https://stackoverflow.com/a/73730116) recommends using `createMemoryRouter`, but that is not exported in `react-router-dom-v5-compat`. Is this intentional? If so, is there a better way? Thanks in advance!

**Comment by ArthurGoupil** [comment]
Hello everyone, 

Any news on this?
Except if I missed something, this prevent me from using `react-router-dom-v5-compat`.

All tests of the (big) repo I'm working on are based on `createMemoryHistory`.

Thanks in advance !
